### ISSUE TITLE
Error running cmakelint: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by BenLeadbetter** [cmake-lint/cmake-lint#36]
I'm using `cmake-lint` in neovim, installed via mason, using the LazyVim distro.

Whenever I make an edit in a cmake file the following error is reported:

```
Error running cmakelint: ENOENT: no such file or directory
```

**Comment by erichlf** [comment]
I also have the exact same problem.

**Comment by pkropachev** [comment]
This is not a problem with `cmake-lint`. LazyVim uses `nvim-lint`, which in turn uses the `cmakelint` binary.
Check https://github.com/LazyVim/LazyVim/pull/1826 and make sure that `cmakelang` and `cmakelint` are installed with Mason.

See https://github.com/mfussenegger/nvim-lint/issues/764.
